### PR TITLE
Update export-teams-content.md

### PR DESCRIPTION
**Pull request by Vikramju**

### CODE REVIEW

**Comment by Vikramju** [comment]
Adding channel message API details to Export APIs

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/export-teams-content.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ appliesto:
...
 
-Teams Export APIs allow you to export 1:1 and group chat messages from Microsoft Teams. If your organization needs to export Microsoft Teams messages, you are able to extract them using Teams Export APIs. *Chat Message* represents an individual chat message within a [channel](https://docs.microsoft.com/graph/api/resources/channel?view=graph-rest-beta) or [chat](https://docs.microsoft.com/graph/api/resources/chat?view=graph-rest-beta). The chat message can be a root chat message or part of a reply thread that is defined by the **replyToId** property in the chat message.
+Teams Export APIs allow you to export 1:1, group chat and channel messages from Microsoft Teams. If your organization needs to export Microsoft Teams messages, you are able to extract them using Teams Export APIs. *Chat Message* represents an individual chat message within a [channel](https://docs.microsoft.com/graph/api/resources/channel?view=graph-rest-beta) or [chat](https://docs.microsoft.com/graph/api/resources/chat?view=graph-rest-beta). The chat message can be a root chat message or part of a reply thread that is defined by the **replyToId** property in the chat message.
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:26]:
```suggestion
Teams Export APIs allow you to export 1:1, group chat, and channel messages from Microsoft Teams. If your organization needs to export Microsoft Teams messages, you are able to extract them using Teams Export APIs. *Chat Message* represents an individual chat message within a [channel](https://docs.microsoft.com/graph/api/resources/channel?view=graph-rest-beta) or [chat](https://docs.microsoft.com/graph/api/resources/chat?view=graph-rest-beta). The chat message can be a root chat message or part of a reply thread that is defined by the **replyToId** property in the chat message.
```

File: Teams/export-teams-content.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ appliesto:
...
 
 Here are some examples on how you can use these export APIs:
 
-- **Example 1**: If you have enabled Microsoft Teams in your organization and want to export all the Microsoft Teams messages to date programmatically by passing the date range for a given user.
-- **Example 2**: If you want to programmatically export all user messages daily by providing a date range. Export APIs can retrieve all the messages created or updated during the given date range.
+- **Example 1**: If you have enabled Microsoft Teams in your organization and want to export all the Microsoft Teams messages to date programmatically by passing the date range for a given user or Team.
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:30]:
```suggestion
- **Example 1**: If you have enabled Microsoft Teams in your organization and want to export all the Microsoft Teams messages to date programmatically by passing the date range for a given user or team.
```

File: Teams/export-teams-content.md [hunk 3]
```diff
@@ -23,12 +23,12 @@ appliesto:
...
+- **Example 2**: If you want to programmatically export all user or Teams messages daily by providing a date range. Export APIs can retrieve all the messages created or updated during the given date range.
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:31]:
```suggestion
- **Example 2**: If you want to programmatically export all user or team messages daily by providing a date range. Export APIs can retrieve all the messages created or updated during the given date range.
```

File: Teams/export-teams-content.md [hunk 4]
```diff
@@ -43,18 +43,23 @@ Here are some examples on how you can use these export APIs:
 
 ## How to access Teams Export APIs
 
-- **Example 1** is a simple query to retrieve all the messages of a user without any filters:
+- **Example 1** is a simple query to retrieve all the messages of a user or Team without any filters:
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:46]:
```suggestion
- **Example 1** is a simple query to retrieve all the messages of a user or team without any filters:
```

File: Teams/export-teams-content.md [hunk 5]
```diff
@@ -43,18 +43,23 @@ Here are some examples on how you can use these export APIs:
...
 
     ```HTTP
     GET https://graph.microsoft.com/beta/users/{id}/chats/allMessages
     ```
+     ```HTTP
+    GET https://graph.microsoft.com/beta/teams/{id}/channels/allMessages
+    ```
 
-- **Example 2** is a sample query to retrieve all the messages of a user by specifying date time filters and top 50 messages:
+- **Example 2** is a sample query to retrieve all the messages of a user or Team by specifying date time filters and top 50 messages:
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:55]:
```suggestion
- **Example 2** is a sample query to retrieve all the messages of a user or team by specifying date time filters and top 50 messages:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**